### ISSUE TITLE
VWA error: cannot find metricbeat template json  

### ISSUE THREAD

**Issue by olafmol** [magneticio/vamp-workflow-agent#13]
```
Registered docker executor on 192.168.99.100
Starting task vamp_workflow-metrics.29bf4305-ed6c-11e6-aa0e-0242202e3455

██╗   ██╗ █████╗ ███╗   ███╗██████╗     ██╗    ██╗ ██████╗ ██████╗ ██╗  ██╗███████╗██╗      ██████╗ ██╗    ██╗
██║   ██║██╔══██╗████╗ ████║██╔══██╗    ██║    ██║██╔═══██╗██╔══██╗██║ ██╔╝██╔════╝██║     ██╔═══██╗██║    ██║
██║   ██║███████║██╔████╔██║██████╔╝    ██║ █╗ ██║██║   ██║██████╔╝█████╔╝ █████╗  ██║     ██║   ██║██║ █╗ ██║
╚██╗ ██╔╝██╔══██║██║╚██╔╝██║██╔═══╝     ██║███╗██║██║   ██║██╔══██╗██╔═██╗ ██╔══╝  ██║     ██║   ██║██║███╗██║
 ╚████╔╝ ██║  ██║██║ ╚═╝ ██║██║         ╚███╔███╔╝╚██████╔╝██║  ██║██║  ██╗██║     ███████╗╚██████╔╝╚███╔███╔╝
  ╚═══╝  ╚═╝  ╚═╝╚═╝     ╚═╝╚═╝          ╚══╝╚══╝  ╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚═╝     ╚══════╝ ╚═════╝  ╚══╝╚══╝

VAMP_KEY_VALUE_STORE_TYPE       : zookeeper
VAMP_KEY_VALUE_STORE_CONNECTION : 192.168.99.100:2181
VAMP_KEY_VALUE_STORE_PATH       : /vamp/workflows/metrics
node  v7.4.0
npm   Exiting: error initializing publisher: Error loading template /usr/local/bin/metricbeat.template.json: open /usr/local/bin/metricbeat.template.json: no such file or directory
Exiting: error initializing publisher: Error loading template /usr/local/bin/metricbeat.template.json: open /usr/local/bin/metricbeat.template.json: no such file or directory
```

**Comment by harmw** [comment]
confirmed 👍 _plus_ we're hitting this:

```
Exiting: error initializing publisher: empty field accessing 'output.elasticsearch.hosts.0' (source:'/usr/local/metricbeat/metricbeat.yml')
```

So:
```

██╗   ██╗ █████╗ ███╗   ███╗██████╗     ██╗    ██╗ ██████╗ ██████╗ ██╗  ██╗███████╗██╗      ██████╗ ██╗    ██╗
██║   ██║██╔══██╗████╗ ████║██╔══██╗    ██║    ██║██╔═══██╗██╔══██╗██║ ██╔╝██╔════╝██║     ██╔═══██╗██║    ██║
██║   ██║███████║██╔████╔██║██████╔╝    ██║ █╗ ██║██║   ██║██████╔╝█████╔╝ █████╗  ██║     ██║   ██║██║ █╗ ██║
╚██╗ ██╔╝██╔══██║██║╚██╔╝██║██╔═══╝     ██║███╗██║██║   ██║██╔══██╗██╔═██╗ ██╔══╝  ██║     ██║   ██║██║███╗██║
 ╚████╔╝ ██║  ██║██║ ╚═╝ ██║██║         ╚███╔███╔╝╚██████╔╝██║  ██║██║  ██╗██║     ███████╗╚██████╔╝╚███╔███╔╝
  ╚═══╝  ╚═╝  ╚═╝╚═╝     ╚═╝╚═╝          ╚══╝╚══╝  ╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═╝╚═╝     ╚══════╝ ╚═════╝  ╚══╝╚══╝

VAMP_KEY_VALUE_STORE_TYPE       : consul
VAMP_KEY_VALUE_STORE_CONNECTION : consul.blaze:8500
VAMP_KEY_VALUE_STORE_PATH       : /vamp/workflows
node  v7.4.0
npm   Exiting: error initializing publisher: empty field accessing 'output.elasticsearch.hosts.0' (source:'/usr/local/metricbeat/metricbeat.yml')
4.0.5
confd 0.11.0
creating confd configuration and template
Exiting: error initializing publisher: empty field accessing 'output.elasticsearch.hosts.0' (source:'/usr/local/metricbeat/metricbeat.yml')
```

**Comment by iamtew** [comment]
@olafmol metricbeat template path fixed under: c2957b2
@harmw that's because you're missing the ES URL, describe under: 4b5b234

**Comment by olafmol** [comment]
reopening, other issues pop up, see new issue
